src/jdk.internal.vm.compiler/share/classes/org.graalvm.compiler.hotspot/src/org/graalvm/compiler/hotspot/GraalHotSpotVMConfig.java

```diff
@@ -253,7 +253,6 @@ public final int logMinObjAlignment() {
 
     public final int stackShadowPages = getFlag("StackShadowPages", Integer.class);
     public final int stackReservedPages = getFlag("StackReservedPages", Integer.class, 0, JDK >= 9);
-    public final boolean useStackBanging = getFlag("UseStackBanging", Boolean.class);
```

Comment:
The line should be modified instead of removal:
```
public final boolean useStackBanging = getFlag("UseStackBanging", Boolean.class, true, JDK < 17);
```
@dougxc please note that it should be upstreamed when GraalVM adopt JDK 17.

Reply:
The good thing is that the VM will fail fail when initializing Graal if the update is not done so there's no way we can overlook it.

Reply:
fixed